From 8c1ac59af79608e07e06fe0b67e887098b95bbfb Mon Sep 17 00:00:00 2001
From: qwang2 <quanyang.wang@windriver.com>
Date: Sat, 9 Jun 2018 14:04:21 +0800
Subject: [PATCH] nxp-imx6: use correct power domain for pd_pci node

In upstream commit 1f1b7ec6714e ("ARM: dts: imx6sx: Add support for PCI power domain"),
the property "power-supply" of node "pd_pci" is set to "reg_pcie". But in freescale's
SDK, the node which is used for power-supply is named as "reg_pcie_phy", so change it
from "reg_pcie" to "reg_pcie_phy".

Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/arm/boot/dts/imx6sx.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 49c9334..8e1f520 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -887,7 +887,7 @@
 					pd_pci: power-domain@3 {
 						reg = <3>;
 						#power-domain-cells = <0>;
-						power-supply = <&reg_pcie>;
+						power-supply = <&reg_pcie_phy>;
 					};
 				};
 			};
-- 
1.7.9.5

